builtin/fstk/ui.lua

```diff
@@ -135,10 +146,10 @@ function ui.update()
 			core.log("warning", "no toplevel ui element "..
 					"active; switching to default")
 			ui.childlist[ui.default]:show()
-			formspec = ui.childlist[ui.default]:get_formspec()
+			formspec = { ui.childlist[ui.default]:get_formspec() }
```

Comment:
```suggestion
			formspec = {ui.childlist[ui.default]:get_formspec()}
```

Reply:
I think the current style is easier to read

Reply:
https://dev.minetest.net/Lua_code_style_guidelines#Spaces
>Spaces are not used around parenthesis, brackets, or curly braces.